README.md

```diff
@@ -53,7 +53,7 @@ image: 'ubuntu@sha256:47f14534bda344d9fe6ffd6effb95eefe579f4be0d508b7445cf77f61a
 **⚠️ Warning!** The README corresponds to the `main` branch of ratchet's
 development, and it may contain unreleased features.
 
-**Pin to specific versions:**
+### Pin to specific versions
```

Comment:
These changes seem unnecessary

Reply:
It allows me to link to the readme using anchors. Used same style for the new doc section added.